gulpfile.js

```diff
@@ -33,11 +33,13 @@ languagesDirs.forEach((langDir) => {
 function clean(done) {
   del([
     "dist/**",
+    "esp_idf_vsc_ext.log",
```

Comment:
Maybe its worth persisting the logs instead of deleting and if developer desire they can delete it themselves ?